### PR TITLE
test: add showcase tests for mixins

### PR DESCRIPTION
**Pull request by parthea**
Towards https://github.com/googleapis/google-cloud-python/issues/16275

Depends on https://github.com/googleapis/gapic-generator-python/pull/2186

### CODE REVIEW

**Comment by parthea** [comment]
[showcase (3.12, showcase_w_rest_async) is failing](https://github.com/googleapis/gapic-generator-python/actions/runs/11004010327/job/30554150445?pr=2175#logs).  This may need to be built on top of #2181

File: tests/system/test_mixins.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    assert response.operations[0].name == "a/pending/thing"
+
+
+def test_delete_operation(echo):
+    echo.delete_operation({"name": "operations/name"})
+
+
+def test_cancel_operation(echo):
+    echo.cancel_operation({"name": "operations/name"})
+
```
Comment (ohmayr) [inline tests/system/test_mixins.py:37]: @parthea what are these two tests doing? Should we add an assert in them?

File: tests/system/test_mixins.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        # First we need to set a policy, before we can get it
+        await async_echo.set_iam_policy(
+            {"resource": "users/user", "policy": {"version": 20240920}}
+        )
+        policy = await async_echo.get_iam_policy(
+            {
+                "resource": "users/user",
+            }
+        )
+        policy.version == 20240920
```
Comment (ohmayr) [inline tests/system/test_mixins.py:126]: missing `assert` keyword.

**Review by ohmayr — APPROVED** [review]

**Review by ohmayr — APPROVED** [review]

File: tests/system/test_mixins.py (hunk 1 above)
Comment (parthea) [reply tests/system/test_mixins.py]: Fixed in https://github.com/googleapis/gapic-generator-python/pull/2175/commits/ff126262eb70936290abc9f1ef6ce345b9cc8d5f

**Pull request merged**